cmd/platform/user.go

```diff
@@ -187,7 +186,7 @@ func changeUserActiveStatus(user *model.User, userArg string, activate bool) err
 		return fmt.Errorf("Can't find user '%v'", userArg)
 	}
 	if user.IsLDAPUser() {
-		return errors.New(utils.T("api.user.update_active.no_deactivate_ldap.app_error"))
+		return errors.New("You can not modify the activation status of AD/LDAP accounts. Please modify through the AD/LDAP server.")
```

Comment:
This was the only string in the file that was being localized. Just grabbed the string from the en file to replace it